README.md

```diff
@@ -1226,31 +1251,24 @@ Async call
 
 And then go to the twitter browser window, keep refreshing it and seeing new tweets still coming.
 
-## Swagger
+## OpenAPI
 
 API management is an important part of modern architectures and comes standard with Ballerina.
 
 It includes ability to use external API gateways or built-in microgateway with security policies such as basic authentication, OAuth, and scopes.
 
-It also knows Swagger - can generate services based on Swagger and export interfaces as Swagger.
+It also knows OpenAPI - can generate services based on OpenAPI and export interfaces as OpenAPI.
 
-To generate Swagger definition of our service simply run:
+To generate OpenAPI definition of our service simply run:
 
 ```
-$ ballerina swagger export demo.bal
-successfully generated swagger definition for input file - demo.bal
-
-$ ls
-ballerina-internal.log  demo.swagger.yaml
-demo.bal                kubernetes
-demo.balx               twitter.toml
+$ ballerina openapi gen-contract hello -i demo.bal
+Note: This is an Experimental tool ship under ballerina hence this will only support limited set of functionality.
```

Comment:
```suggestion
Note: This is an experimental tool shipped under ballerina hence this will only support a limited set of functionality.
```

Reply:
That is the result we are getting when running the command. We need to fix it

Reply:
Created an issue https://github.com/ballerina-platform/ballerina-lang/issues/18990